cmd/rt-conf/main.go

```diff
@@ -23,7 +23,10 @@ func main() {
 
 func run(args []string) error {
 	envConfigFile := os.Getenv("CONFIG_FILE")
-	envVerbose, _ := strconv.ParseBool(os.Getenv("VERBOSE"))
+	envVerbose, err := strconv.ParseBool(os.Getenv("VERBOSE"))
+	if err != nil {
+		log.Printf("Warning: failed to parse VERBOSE env var, defaulting to false: %v", err)
+	}
```

Comment:
[Any other value returns an error.](https://pkg.go.dev/strconv#:~:text=Any%20other%20value%20returns%20an%20error.)

They do not define the default value as false when an error occurred. Maybe better to actually set it to false.

```suggestion
	envVerbose, err := strconv.ParseBool(os.Getenv("VERBOSE"))
	if err != nil {
		log.Printf("Warning: failed to parse VERBOSE env var, defaulting to false: %v", err)
		envVerbose = false
	}
```

Reply:
Note: If you do the above, this code should also change to export only if the snap option is set: https://github.com/canonical/rt-conf/blob/1ce23b4d678df328c89f3b6543d1b68060ebbf2c/snap/local/bin/export-env.sh#L4C1-L4C38